src/snowflake/snowpark/dataframe.py

```diff
@@ -481,7 +482,9 @@ def collect(
 
         Args:
             statement_params: Dictionary of statement level parameters to be set while executing this action.
-            block: A bool value indicating whether this function will wait until the result is available. When it is ``False``,  this function executes the underlying queries of the dataframe asynchronously and returns an :class:`AsyncJob`.
+            block: A bool value indicating whether this function will wait until the result is available.
```

Comment:
Do we need to warn users that this parameter is experimental in code like deprecated?

Reply:
yes, see next two lines.